### PR TITLE
Handle request-start header in microseconds and nanoseconds

### PR DESCRIPTION
**Pull request by adamlogic**
Not entirely sure wither Render uses microseconds or nanoseconds, so we'll handle both.

### CODE REVIEW

File: judoscale-ruby/lib/judoscale/request_metrics.rb [hunk 1]
```diff
@@ -20,15 +24,23 @@ def started_at
       if @request_start_header
         # There are several variants of this header. We handle these:
         #   - whole milliseconds (Heroku)
+        #   - whole microseconds (???)
```
Comment (jon-sully) [inline judoscale-ruby/lib/judoscale/request_metrics.rb:27]: Lol what?

**Review by jon-sully — COMMENTED** [review]
I like the cutoff hierarchy — elegant solution 👍

File: judoscale-ruby/lib/judoscale/request_metrics.rb (hunk 1 above)
Comment (adamlogic) [reply judoscale-ruby/lib/judoscale/request_metrics.rb:27]: Prior to this PR, my code comment said "nanoseconds", but my implementation handled "microseconds". I'm not even sure which one Render uses. I can't find it documented anywhere.

Comment (jon-sully) [reply judoscale-ruby/lib/judoscale/request_metrics.rb:27]: Well dang.

[image](https://stackoverflow.com/a/24106694/7552158)

But I think we're well within safe constraints. Interesting they don't mention it anywhere though. Maybe they don't anticipate people using that header

**Pull request merged**